block/manager.go

```diff
@@ -328,3 +330,22 @@ func (m *Manager) setDA(daconfig string, dalcKV store.KV, logger log.Logger) err
 	m.Retriever = retriever
 	return nil
 }
+
+func (m *Manager) gossipPendingBlocks(ctx context.Context) {
```

Comment:
seems like we're not updating the LastGossipedHeight here or in the `gossipBlock` function.

Reply:
updated in gossipBlock